dbt_subprojects/tokens/models/transfers_and_balances/abstract/tokens_abstract_transfers.sql

```diff
@@ -4,7 +4,7 @@
      partition_by = ['block_month'],
      materialized = 'incremental',
      file_format = 'delta',
-     incremental_strategy = 'merge',
+     incremental_strategy = 'append',
```

Comment:
if we do append without filtering on `> max(timestamp) from {{this}}`, we will start introducing duplicates during the incremental updates.

If we do a self referential filter we can't update already written columns and might have gaps with null values do to unequal data recency. We need to use `merge`